config/samples/dataplane_v1beta1_openstackdataplanenodeset.yaml

```diff
@@ -42,15 +42,6 @@ spec:
       ansibleUser: cloud-admin
```

Comment:
Can we default this in common.go and then drop it from the samples?

Reply:
Sounds good. Done in the latest iteration.